logstash-core/lib/logstash/runner.rb

```diff
@@ -115,6 +115,11 @@ class LogStash::Runner < Clamp::StrictCommand
     :attribute_name => "http.port",
     :default => LogStash::SETTINGS.get_default("http.port"), &:to_i
 
+  option ["--http.environment"], "HTTP_ENVIRONMENT",
```

Comment:
Is it possible to add a file-only setting? I'm fine with that but it has to be able to be set in the runner instantiation.